pkg/commands/image/list_filter.go

```diff
@@ -0,0 +1,145 @@
+package image
+
+import (
+	corev1alpha1 "github.com/pivotal/kpack/pkg/apis/core/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"regexp"
+	"strings"
+
+	"github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
+)
+
+const (
+	filterKindBuilder        = "builder"
+	filterKindClusterBuilder = "clusterbuilder"
+	filterKindLatestReason   = "latest-reason"
+	filterKindStatus         = "ready"
+)
+
+type filter struct {
+	kind   string
+	values []string
+}
+
+func Filter(images *v1alpha1.ImageList, flags []string) *v1alpha1.ImageList {
```

Comment:
Can we avoid exporting this?

Reply:
It's exported for the sake of testing.. An alternative could've been to test all that functionality through _list_test.go_, but then it get too bloated.

Reply:
We have been taking the approach that to avoid unnecessary exports the `list_filter` test should be in the image package instead of the image_test package.

Reply:
Sure.